### PR TITLE
Change the way update command checks the node status

### PR DESCRIPTION
**Pull request by a1exsh**
* Don't rely on DownEndpointCount: this is known to be inaccurate.
* Add a warning for the user to stop and check monitoring before update of
  each node.

### CODE REVIEW

File: planb/update_cluster.py [hunk 1]
```diff
@@ -533,6 +527,9 @@ def update_cluster(options: dict):
         # TODO: user should hit Ctrl-c to cancel everything
         # don't ask again if resuming after crash
         if len(instances) > 1:
+            logger.warn("-----------------------------------------------")
+            logger.warn("!!! Check your monitoring before proceeding !!!")
```
Comment (slitsche) [inline planb/update_cluster.py]: Should we be more specific? Like "Do all nodes agree on no node is down?"

Comment (a1exsh) [reply planb/update_cluster.py]: Well, I think it is more general: e.g. if the latencies were hurt or something else the user cares about.

**Comment by slitsche** [comment]
👍

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**